stable/ark/templates/deployment.yaml

```diff
@@ -35,6 +35,23 @@ spec:
             - /ark
           args:
             - server
+          {{- with .Values.configuration }}
+            {{- with .metricAddress }}
+            - --metrics-address={{ . }}
+            {{- end }}
+            {{- with .backupSyncPeriod }}
+            - --backup-sync-period={{ . }}
+            {{- end }}
+            {{- with .resticTimeout }}
+            - --restic-timeout={{ . }}
+            {{- end }}
+            {{- with .restoreOnlyMode }}
+            - --restore-only={{ . }}
```

Comment:
This won't work if restoreOnlyMode=false is specified. The --restore-only parameter does not accept a value but defaults to true if specified. I would suggest changing this to the following:

`            {{- if .restoreOnlyMode }}`
`            - --restore-only`
`            {{- end }}`

Reply:
changed